doc/source/train/user_guide.rst

```diff
@@ -977,6 +977,30 @@ the disk that from which your script was executed from.
     # View the PyTorch Profiler traces.
     $ open http://localhost:6006/#pytorch_profiler
 
+
```

Comment:
Can you add a section bookmark above this? This would help if we want to easily reference this section from other parts of the docs.

Reply:
Fixed